app/src/androidTest/java/TestPlan.kt

```diff
@@ -17,6 +19,76 @@ class TestPlan : BaseTestPlan() {
         homePage.recyclerView.clickRecyclerViewItem(1)
     }
 
+    @Test
+    fun checkNoteScreenHeadingEditText() {
+        clickBottomToolbarButton(HOME_PAGE_BOTTOM_BAR_ADD_NOTE_BUTTON_NUMBER)
```

Comment:
I think you can encapsulate all these operations inside *Screen. *Plan class should describe WHAT should be tested, but *Screen class should describe HOW it should work. E.g. (pseudocode)
```
homePage.execute {
    clickAddNoteButton()
    verifySomeSpecificInputIs(R.string.format_hint_heading)
    swipeNotesList()
}
```
`HomePage` should describe HOW e.g. clickAddNoteButton should work, what index/id/position it has etc.